src/test_l1_hdmi_cec_driver.c

```diff
@@ -68,6 +68,7 @@
 
 #include <ut.h>
 #include "ut_log.h"
+#include "ut_kvp_profile.h"
```

Comment:
techically include "file.h" means search the local local directory then Path, where include <file.h>, means look in the path.

so #include <file.h> is more correct.

Reply:
Updated the lines
#include "ut_log.h"
#include "ut_kvp_profile.h"